commons/src/main/java/gov/cms/qpp/conversion/DocumentationReference.java

```diff
@@ -0,0 +1,19 @@
+package gov.cms.qpp.conversion;
+
+public enum DocumentationReference {
+	CLINICAL_DOCUMENT(19),
+	MEASURE_IDS(88);
+
+	private static final String BASE_PATH = "https://ecqi.healthit.gov/system/files/eCQM_QRDA_EC-508_0.pdf#page=";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)

Reply:
No